timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -217,6 +217,40 @@ types:
       LeaderTimes:
         fields:
           leaderTimes: map<Namespace, LeaderTime>
+      TimestampName:
+        alias: string
+        safety: safe
+      NamedMinTimestampLeaseRequest:
+         fields:
+           timestampName: TimestampName
+           requestId:
+             type: uuid
+             safety: safe
+           numFreshTimestamps:
+             type: integer
+             safety: safe
+      NamedMinTimestampLeaseRequests:
+         alias: list<NamedMinTimestampLeaseRequest>
+      MultiClientNamedMinTimestampLeaseRequest:
+         alias: map<Namespace, NamedMinTimestampLeaseRequests>
+      NamedMinTimestampLeaseResponse:
+         fields:
+           minLeased: Long
+           lock: LockToken
+           lease: Lease
```

Comment:
LeasedLockToken is not exposed where I need it and I am not sure how safe for serde it is. This work ok and I got familiar about how the flow works to wire things correctly